README.md

```diff
@@ -15,6 +15,7 @@ Each example contains a README.md with an explanation about the solution and it'
 | Example | Runtime  |
 |:--------------------------- |:-----|
 | [`fractal-terrain-generation`](fractal-terrain-generation) <br/> Render terrain in a browser with a backbend implemented in Serverless functions | nodeJS |
+| [`lambda-on-binaris`](lambda-on-binaris) <br/> Run simple AWS Lambda functions while gaining the benefits of the Binaris platform | nodeJS |
```

Comment:
style: maybe lambda-wrapper ? (lambda on binaris is a bit weird. maybe it's just me)